test/dynamo/test_misc.py

```diff
@@ -9309,7 +9309,7 @@ def test_shape_env_equal_create_symbolic_sizes_strides_storage_offset(self):
   >  Left: {0: 0, 1: 1, 2: s1, 3: s0}
   > Right: {0: 0, 1: 1}
 ==> var_to_range: values don't match.
-  >  Left: {s0: ValueRanges(lower=2, upper=9223372036854775806, is_bool=False, is_int=True, is_float=False), s1: ValueRanges(lower=2, upper=9223372036854775806, is_bool=False, is_int=True, is_float=False)}
+  >  Left: {s0: ValueRanges(lower=2, upper=IntInfinity(), is_bool=False, is_int=True, is_float=False), s1: ValueRanges(lower=2, upper=IntInfinity(), is_bool=False, is_int=True, is_float=False)}
```

Comment:
As a follow-up (not in this PR) you could probably print inf and -inf as oo and -oo (or inf / -inf) in the VR.__repr__.

Reply:
I'm actually not entirely sure how sympy.oo customizes its printing lol

Reply:
It's done at https://github.com/sympy/sympy/blob/master/sympy/printing/str.py#L177-L178

You can define
```
def _sympystr(self, printer):
         return "int_oo"
```
in the object itself for printing.

This is done so that sympy can have different printers. (python, c, etc)